Jellyfin.Api/Controllers/DevicesController.cs

```diff
@@ -111,28 +114,34 @@ public async Task<ActionResult> UpdateDeviceOptions(
     }
 
     /// <summary>
-    /// Deletes a device.
+    /// Deletes devices.
     /// </summary>
-    /// <param name="id">Device Id.</param>
+    /// <param name="id">Optional. Device Id.</param>
+    /// <param name="ids">Device Ids.</param>
     /// <response code="204">Device deleted.</response>
     /// <response code="404">Device not found.</response>
-    /// <returns>A <see cref="NoContentResult"/> on success, or a <see cref="NotFoundResult"/> if the device could not be found.</returns>
+    /// <returns>A <see cref="NoContentResult"/> on success, or a <see cref="NotFoundResult"/> if a device could not be found.</returns>
     [HttpDelete]
     [ProducesResponseType(StatusCodes.Status204NoContent)]
     [ProducesResponseType(StatusCodes.Status404NotFound)]
-    public async Task<ActionResult> DeleteDevice([FromQuery, Required] string id)
+    public async Task<ActionResult> DeleteDevice([FromQuery, ParameterObsolete] string? id, [FromQuery, ModelBinder(typeof(CommaDelimitedArrayModelBinder))] string[] ids)
```

Comment:
IIRC we don't use `CommaDelimitedArrayModelBinder` for new parameters, that's just for old API's

Reply:
What should be used instead?

Reply:
You don't need to specify a model binder, the comma-delimited binder is for legacy support

Reply:
So instead, we should delimit the ids manually? Searching for `] string[]`, all instances have some kind of model binder. Testing with a comma delimited string without the model binder returns the entire parameter for me.

Or, am I missing something in .NET that does this for us?

Reply:
You would change it from `[FromQuery, ModelBinder(typeof(CommaDelimitedArrayModelBinder))] string[] ids` to just `[FromQuery] string[] ids`. Then to supply ids you use `?ids=id1&ids=id2&ids=id3` etc.